src/modules/addimage.js

```diff
@@ -746,13 +758,22 @@ import { atob, btoa } from "../libs/AtobBtoa.js";
     }
   });
 
+  /**
+   * Possible parameter for addImage, an RGBA buffer with size.
+   *
+   * @typedef {Object} RGBAData
+   * @property {Uint8ClampedArray} data - Single dimensional array of RGBA values. For example from canvas getImageData.
+   * @property {number} width - Image width as the data does not carry this information in itself.
+   * @property {number} height - Image height as the data does not carry this information in itself.
+   */
+
```

Comment:
We also need to add this to the typings file + add a typings test.

Reply:
Alright. I will be on holiday for the next week and will return to this after 2nd of April.